### ISSUE TITLE
Missing fields are deserialized to nulls

### ISSUE THREAD

**Issue by hvester** [panesofglass/FSharp.Data.JsonSchema#11]
If json does not contain a field corresponding to a property then I would assume the deserialization to fail instead of leaving the value of the property as null. I'm wondering if this is intentional or a bug?

```fsharp
#r "nuget: FSharp.Data.JsonSchema"

open FSharp.Data

type Foo = { Bar : string }

Json.Deserialize<Foo>("{}")
// val it : Foo = { Bar = null }
```

Seems to be caused by `IgnoreNullValues=true` in the [DefaultOptions](https://github.com/panesofglass/FSharp.Data.JsonSchema/blob/master/src/FSharp.Data.JsonSchema/Serializer.fs#L15).

**Comment by hvester** [comment]
I tested and it works as expected.

**Comment by panesofglass** [comment]
I've released this update in 2.0.0
